website/content/v1.9/reference/configuration/pci/pcirebindconfig.md

```diff
@@ -0,0 +1,35 @@
+---
+description: PCIRebindConfig allows to configure PCI driver rebinds.
+title: PCIRebindConfig
+---
+
+<!-- markdownlint-disable -->
+
+
+
+
+
+
+
+
+
+{{< highlight yaml >}}
+apiVersion: v1alpha1
+kind: PCIRebindConfig
+name: ixgbe # Name of the config document.
+vendorDeviceID: 0000:04:00.00 # PCI device vendor and device ID.
```

Comment:
Well, not vendor/device ID but device path obtained from lspci or ID from `PCIDevices.hardware.talos.dev`

Reply:
I guess calling it pci id or bdf is the right term then